### PR TITLE
Switch to anonymous components

### PR DESCRIPTION
**Pull request by OBrezhniev**
* Unification of circuits.
* Switch to anonymous components.
* Removed Auth (v1) circuits & tests.
* Some templates got enabled input signal.
* EdDSAPoseidonVerifier tests. 
* credentialAtomicQueryV3 - unified sig&mtp circuit for offchain usage.

### CODE REVIEW

**Review by AndriianChestnykh — APPROVED** [review]

File: circuits/lib/utils/claimUtils.circom [hunk 1]
```diff
@@ -15,11 +15,11 @@ template getClaimSubjectOtherIden() {
 
     // get header flags from claim.
     component header = getClaimHeader();
-    for (var i=0; i<8; i++) { header.claim[i] <== claim[i]; }
+    header.claim <== claim;
```
Comment (ilya-korotya) [inline circuits/lib/utils/claimUtils.circom:18]: I think we can call getClaimHeader:
```
var claimType claimFlags ...
(claimType, claimFlags) = getClaimHeader()(claim);
```
I found this in the documentation: https://docs.circom.io/circom-language/anonymous-components-and-tuples/#syntax-and-semantics-of-anonymous-components

File: circuits/lib/utils/claimUtils.circom [hunk 2]
```diff
@@ -1,4 +1,4 @@
-pragma circom 2.0.0;
+pragma circom 2.1.1;
```
Comment (ilya-korotya) [inline circuits/lib/utils/claimUtils.circom:1]: `2.1.1` is not the latest version. Check latest version, please.

**Review by ilya-korotya — APPROVED** [review]

File: circuits/lib/utils/claimUtils.circom [hunk 3]
```diff
@@ -15,11 +15,11 @@ template getClaimSubjectOtherIden() {
 
     // get header flags from claim.
     component header = getClaimHeader();
-    for (var i=0; i<8; i++) { header.claim[i] <== claim[i]; }
+    header.claim <== claim;
 
     // get subject location from header flags.
     component subjectLocation = getSubjectLocation();
```
Comment (ilya-korotya) [inline circuits/lib/utils/claimUtils.circom:21]: We can use anonym component

File: circuits/lib/utils/claimUtils.circom [hunk 4]
```diff
@@ -48,11 +48,11 @@ template getClaimMerklizeRoot() {
 
     // get header flags from claim.
     component header = getClaimHeader();
-    for (var i=0; i<8; i++) { header.claim[i] <== claim[i]; }
+    header.claim <== claim;
 
     // get subject location from header flags.
     component merklizeLocation = getMerklizeLocation();
```
Comment (ilya-korotya) [inline circuits/lib/utils/claimUtils.circom:54]: We can use anonym component

File: circuits/lib/utils/claimUtils.circom (hunk 1 above)
Comment (OBrezhniev) [reply circuits/lib/utils/claimUtils.circom:18]: Not working for `claimFlags[32]`

**Review by ilya-korotya — APPROVED** [review]

**Pull request merged**